torch/utils/hipify/hipify_python.py

```diff
@@ -72,6 +72,39 @@ class bcolors:
     BOLD = '\033[1m'
     UNDERLINE = '\033[4m'
 
+class GeneratedFileCleaner:
```

Comment:
The naming here is a little weird, because from the context of this script itself `hipify_python.py` these aren't intermediates at all; they're the output of the hipification process. So to me this evokes more of a, "hey, please tell me what output files you're actually going to generate here!" similar to the FileManager abstraction we have in our other codegen scripts. I don't really want to bikeshed here, but a comment at this class talking about the intent (specifically for the jit extensions case) would help future refactorers a bit, I think.

Reply:
Right.